hpx/hpx_distributed.cc

```diff
@@ -0,0 +1,380 @@
+/* Copyright 2020 Stanford University
```

Comment:
It's ok to put your own names here (or whoever the copyright holder is). It's just the license that we want to match.

Reply:
I added the student names.